File: fontes/api/CarePlusAPI/CarePlusAPI/Models/Usuario/UsuarioModel.cs

using CarePlusAPI.Models.Perfil;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace CarePlusAPI.Models.Usuario
{
    [ExcludeFromCodeCoverage]
    public class UsuarioModel
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string NomeUsuario { get; set; }
        public char Ativo { get; set; }
        public List<PerfilModel> UsuarioPerfil { get; set; }
    }
}

File: fontes/cms/src/models/usuario/usuario.model.ts
import { CareplusPerfilModel } from './../careplus-perfil/careplus-perfil.model';
export class UsuarioModel {
  public constructor(init?: Partial<UsuarioModel>) {
    Object.assign(this, init);
  }
  id: number;
  nome: string;
  ativo:string;
  nomeUsuario: string;
  SenhaHash: string;
  SenhaSalt: string;
  usuarioPerfil: any;
}

